torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -770,6 +770,7 @@ class MKLDNNSubgraphSlicer {
       // conversions. from initial testing including it speeds up models
       case aten::max_pool2d:
       case aten::max_pool3d:
+      case aten::adaptive_avg_pool2d:
```

Comment:
Do you mind adding a few more ops while you're at it?  `adaptive_avg_pool3d`, `tanh`, `avg_pool2d, `avg_pool3d` ?

Reply:
will do